modules/users/client/views/settings/manage-social-accounts.client.view.html

```diff
@@ -1,47 +1,47 @@
-<section class="row">
-  <h3 class="col-md-12 text-center" ng-show="vm.hasConnectedAdditionalSocialAccounts()">Connected social accounts:</h3>
+<md-content>
+  <h1 class="md-display" ng-show="vm.hasConnectedAdditionalSocialAccounts()">Connected social accounts:</h1>
   <div class="col-md-12 text-center">
     <div ng-repeat="(providerName, providerData) in vm.user.additionalProvidersData" class="social-account-container">
-      <img ng-src="/dist/img/users/client/img/buttons/{{providerName}}.png">
+      <img ng-src="/modules/users/client/img/buttons/{{providerName}}.png">
```

Comment:
I think these need to be /dist/img.  /modules is not longer accessible by the web.  Only the public folder